extensions/ql-vscode/src/model-editor/model-alerts/model-alerts-view.ts

```diff
@@ -0,0 +1,67 @@
+import { ViewColumn } from "vscode";
+import type { WebviewPanelConfig } from "../../common/vscode/abstract-webview";
+import { AbstractWebview } from "../../common/vscode/abstract-webview";
+import { assertNever } from "../../common/helpers-pure";
+import { telemetryListener } from "../../common/vscode/telemetry";
+import type {
+  FromDataFlowPathsMessage,
+  ToDataFlowPathsMessage,
+} from "../../common/interface-types";
+import type { App } from "../../common/app";
+import { redactableError } from "../../common/errors";
+import { extLogger } from "../../common/logging/vscode";
+import { showAndLogExceptionWithTelemetry } from "../../common/logging";
+
+export class ModelAlertsView extends AbstractWebview<
+  ToDataFlowPathsMessage,
```

Comment:
Can we define new types instead of reusing the `DataFlowPaths` types

Reply:
Oooops, I thought I'd already fixed that! Thanks 🤦🏽